extension/app/src/components/DropzoneContainer.tsx

```diff
@@ -0,0 +1,61 @@
+import { DropzoneOverlay } from "@dust-tt/sparkle";
```

Comment:
(Not reviewing this one since same as front version!)

Reply:
yes, same